CCSA_ChatApp.Authentication/Services/AdminHandler.cs

```diff
@@ -8,13 +8,13 @@ public class AdminHandler : AuthorizationHandler<AdminRequirement>
     {
         protected override async Task HandleRequirementAsync(AuthorizationHandlerContext context, AdminRequirement requirement)
         {
```

Comment:
I really don't know what you're doing here

Reply:
Please could you drop resources on the group that can help me understand this AuthorizationHandlerContext and co?